packages/neuron-ui/src/components/GeneralSetting/index.tsx

```diff
@@ -37,6 +37,7 @@ const UpdateDownloadStatus = ({
     return (
       <Dialog
         show={show}
+        // @ts-ignore
```

Comment:
`ts-ignore` should not be added unless we can only do this, and add it with annotation